packages/chat-component/src/core/parser/index.ts

```diff
@@ -110,8 +117,7 @@ export async function parseStreamedMessages({
 
     visit();
   }
-
-  return streamedMessageRaw.join('');
+  return streamedMessageRaw.join(''), thoughtProcess;
```

Comment:
I'm not what's the intent here? I you want to return both it should be in an array `[streamedMessageRaw.join(''), thoughtProcess]`, as currently only `thoughtProcess` is returned?

Reply:
Yep, good catch. Initially I was going to wrap both in an object, but streamedMessageRaw is not used, so I was going to delete it all together. I left it for later to decide, and I simply forgot :)